src/main/resources/jenkins/plugins/slack/workflow/SlackSendStep/help-notifyCommitters.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    <p>Notify committers via direct message in addition to specified channels, requires using a bot user</p>
```

Comment:
It would be good to be able to not send to a channel and only DM in certain circumstances, e.g. on a pull request we don't care that the build failed only the submitter, 
Do you think their would need to be code changes to support this? or would setting channel to an empty string on a PR work?

I'm conscious that `null` quite likely won't work because their might be global config for a default channel, potentially an ignoreGlobalConfig flag may be needed.

Reply:
I have a follow-up PR that adds a new pipeline step for looking up a slack userId (by email). In that case you could use the result of that pipeline step to override the channel.